### PR TITLE
Document changes to the sign-in experience

### PR DESCRIPTION
**Pull request by karl-cardenas-coding**
This PR documents the new UI user authentication flow.
💻 [Preview](https://deploy-preview-1100--docs-spectrocloud.netlify.app/user-management/user-authentication)

CleanShot 2023-01-31 at 14 45 42

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1100#" title="Author self-approved">karl-cardenas-coding</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-2/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by amarjeet27 — APPROVED** [review]

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1100#pullrequestreview-1278108996" title="Approved">amarjeet27</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1100#" title="Author self-approved">karl-cardenas-coding</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-2/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/08-user-management/1-user-authentication.md [hunk 1]
```diff
@@ -6,9 +6,51 @@ icon: ""
...
+
+Starting with Palette 3.2, the user sign-in flow can be different depending on how you created your Palette account. If you made your user with a username and password, then you may be promoted to select the organization you wish to log in to. If you are a member of a single organization, then you will not be prompted for an organization selection.
+
+If you created an account through SSO and are a member of different organizations, then you must first select the organization name you wish to log in to. Click on the **Sign in to your organization** button for the option to specify the organization name. If you need help remembering the organization name, click on the **Forgot your organization name?** button and provide your email address to receive an email containing your organization name and its login URL.
+
+<br />
+
+<InfoBox>
+
+If you are a Palette Enterprise user, use the custom Palette URL for an optimized log in experience and avoid specifying the organization name.
```
Comment (ritawatson) [inline content/docs/08-user-management/1-user-authentication.md:50]:
```suggestion
If you are a Palette Enterprise user, use the custom Palette URL for an optimized login experience and avoid specifying the organization name. 
```

File: content/docs/08-user-management/1-user-authentication.md [hunk 2]
```diff
@@ -6,9 +6,51 @@ icon: ""
...
+<!-- vale on -->
+## Account Sign Up
+
+You can sign up for a Palette SaaS account by visiting [Palette](https://console.spectrocloud.com) or an Enterprise Palette account under your organization by using your organization's custom Palette URL.
+
+When you create an account, you can create a username and password or create the account through a third-party identity provider, GitHub, Google, or other OIDC providers that are enabled for your organization. For Palette SaaS, GitHub and Google are automatically enabled for SSO integration.
+
+## Sign In Flow
+
+Starting with Palette 3.2, the user sign-in flow can be different depending on how you created your Palette account. If you made your user with a username and password, then you may be promoted to select the organization you wish to log in to. If you are a member of a single organization, then you will not be prompted for an organization selection.
```
Comment (ritawatson) [inline content/docs/08-user-management/1-user-authentication.md:42]:
```suggestion
Starting with Palette 3.2, the user sign-in flow can be different depending on how you created your Palette account. If you created your user with a username and password, then you may be promoted to select the organization you wish to log in to. If you are a member of a single organization, then you will not be prompted for an organization selection.
```

**Review by ritawatson — APPROVED** [review]
Just 2 minor comments. I saw you fixed 'tooken'. :)

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1100#pullrequestreview-1278108996" title="Approved">amarjeet27</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1100#" title="Author self-approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1100#pullrequestreview-1278112775" title="Approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-2/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/08-user-management/1-user-authentication.md [hunk 3]
```diff
@@ -6,9 +6,51 @@ icon: ""
...
+<!-- vale on -->
+## Account Sign Up
+
+You can sign up for a Palette SaaS account by visiting [Palette](https://console.spectrocloud.com) or an Enterprise Palette account under your organization by using your organization's custom Palette URL.
+
+When you create an account, you can create a username and password or create the account through a third-party identity provider, GitHub, Google, or other OIDC providers that are enabled for your organization. For Palette SaaS, GitHub and Google are automatically enabled for SSO integration.
+
+## Sign In Flow
+
+Starting with Palette 3.2, the user sign-in flow can be different depending on how you created your Palette account. If you created your user with a username and password, then you may be promoted to select the organization you wish to log in to. If you are a member of a single organization, then you will not be prompted for an organization selection.
```
Comment (karl-cardenas-coding) [inline content/docs/08-user-management/1-user-authentication.md:42]:
```suggestion
Starting with Palette 3.2, the user sign-in flow can be different depending on how you created your Palette account. If you created your user with a username and password, then you may be prompted to select the organization you wish to log in to. If you are a member of a single organization, then you will not be prompted for an organization selection.
```

**Pull request merged**